packages/backend-core/src/db/couch/DatabaseImpl.ts

```diff
@@ -246,6 +246,35 @@ export class DatabaseImpl implements Database {
     })
   }
 
+  async bulkRemove(documents: Document[], opts?: { silenceErrors?: boolean }) {
+    const response: Nano.DocumentBulkResponse[] = await this.performCall(db => {
+      return () =>
+        db.bulk({
+          docs: documents.map(doc => ({
```

Comment:
Is this atomic, or can we get an inconsistent state?

Reply:
It isn't atomic no, but there isn't really anyway to get around that.